doc/ref_impl/cntt-ri2/chapters/chapter01.md

```diff
@@ -1,4 +1,4 @@
-﻿# 1. Overview
+# 1. Overview
 
 ![State](../figures/bogo_dfp.png) <!-- width="35" -->
```

Comment:
I thought we agreed to delete the bogometers

Reply:
@pgoyal01 we did. They have been removed in PR #2656 which implements the rst conversion for RI2.